openvpn/build/gentoo-config-0.9.8

```diff
@@ -0,0 +1,194 @@
+#!/bin/sh
+# Copyright 1999-2005 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
```

Comment:
me have lied with the cla on this point - this is the original: https://github.com/OpenVPN/openvpn-build/blob/master/generic/gentoo.config-0.9.8